shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/visitor/impl/MySQLDALVisitor.java

```diff
@@ -144,15 +151,31 @@ public ASTNode visitShowOther(final ShowOtherContext ctx) {
     @Override
     public ASTNode visitSetVariable(final SetVariableContext ctx) {
         SetStatement result = new SetStatement();
-        if (null != ctx.variable()) {
-            result.setVariable((VariableSegment) visit(ctx.variable()));
+        
+        List<VariableExprContext> variableExprContextList = ctx.variableExpr();
+        if (null != variableExprContextList) {
+            List<VariableExpr> variableExprList = new ArrayList<VariableExpr>(variableExprContextList.size());
+            for (int i = 0; i < variableExprContextList.size(); i++) {
+                VariableExprContext variableExprContext = variableExprContextList.get(i);
+                VariableSegment variableSegment = (VariableSegment) visitVariable(variableExprContext.variable());
+                VariableValueSegment variableValueSegment = (VariableValueSegment) visitVariableValue(variableExprContext.variableValue());
+                String scopeType = variableExprContext.variable().scope() == null ? null : variableExprContext.variable().scope().getText();
```

Comment:
`scopeType` is inconsistent with rule name `scope`

Reply:
done.use scope